### ISSUE TITLE
[Feature Request] ability to filter recents history

### ISSUE THREAD

**Issue by PointVoider** [Jays2Kings/tachiyomiJ2K#817]
### Why/User Benefit/User Problem
(explain why this feature should be added)
This is useful for someone that always forget to add manga to their library after done reading it.
I've done this several times, first i will read the manga wether i will save it my library or not. If it's good, I will keep reading it until i completed all the chapter then forgot to add it to my library.
### What/Requirements
(explain how this feature would behave)
It should filter to show all my history that are not on my library

**Comment by Jays2Kings** [comment]
This is outdated, but the history tab in recents, or the grouped/all section (if you enable show read chapters) show history of manga not in your library

**Comment by PointVoider** [comment]
Yeah, but it's also shows manga that are in my library which make it harder to find manga that are not in my library

**Comment by Jays2Kings** [comment]
I'll reopen since I misunderstood what the request was, still not sure about actually adding yet
